src/pages/overview/limits.md

```diff
@@ -98,11 +88,11 @@ Paid credentials enable a greater processing quota and require entering
 into a separate written agreement. To learn more, please see our
 [pricing page](/document-services/pricing/#main) for purchase options.
 
-### Usage limits
+### Usage Limits
 
 There are several usage limits that apply to PDF Services API and its underlying Operations based on one initial endpoint request. Files submitted for processing that exceed usage limits below will fail and result in an error message.
 
-| Usage limit                                                     | Value       |
+| USAGE LIMIT                                                     | VALUE       |
```

Comment:
Page limit (Extract and ccessibility Auto-Tag)*
^^ (A)ccessibility here A is missing

Reply:
Page limit(Scanned - Extract and Accessibility Auto-Tag )* change this to Page limit (Scanned - Extract and Accessibility Auto-Tag )*

Reply:
100 RPM* (Enterprise) 

25 RPM (Free Trial) 

Please enter a new line between these two. Also add two (**) in above inplace of one to map to "100 RPM* (Enterprise) 25 RPM (Free Trial) "